python/python/bystro/vcf_utils/tests/test_sim_data_generator.py

```diff
@@ -0,0 +1,35 @@
+"""Test simulated data."""
+
+import pandas as pd
+from bystro.vcf_utils.sim_data_generator import (
+    generate_random_vcf_index, 
+    generate_simulated_vcf
+)
+
+HEADER_COLS = ["#CHROM", "POS", "ID", "REF", "ALT", "QUAL", "FILTER", "INFO", "FORMAT"]
+
+
+def test_generate_random_vcf_index():
+    """Test expected index components have correct parameters."""
+    random_vcf_index = generate_random_vcf_index()
+    assert isinstance(random_vcf_index, str)
+    parts = random_vcf_index.split(":")
+    assert len(parts) == 4
+    assert parts[0].isdigit()
+    assert 1 <= int(parts[0]) <= 22
+    assert parts[1].isdigit()
+    assert 1 <= int(parts[1]) <= 1000000
+    assert parts[2] in ["A", "T", "C", "G"]
+    assert parts[3] in ["A", "T", "C", "G"]
+    assert parts[2] != parts[3]
+
+
+def test_generate_simulated_vcf():
+    """Test generate_simulated_vcf."""
+    num_samples = 10
+    num_vars = 10
+    simulated_vcf, simulated_indices = generate_simulated_vcf(num_samples, num_vars)
+    assert isinstance(simulated_vcf, pd.DataFrame)
+    assert isinstance(simulated_indices, list)
+    assert len(simulated_indices) == num_vars
```

Comment:
See above comment: because simulated_indices is reduplicating info from the df, we have to write this check-- if we just return the df then there's no way for this to go wrong because it's impossible to construct a df with a differing number of rows per column.  

This is a particular case of the general principle that whenever we have two lists (or list-like collections) of corresponding data, it's better to express that as a list of tuples than two separate lists, because in the latter case we can't even write the error of the lists not being of the same length.  There are certain exceptions where the contrary convention has the weight of tradition behind it (like writing `X, y` for the features and labels of a dataset in ML) but it's a useful habit to keep in mind.